### ISSUE TITLE
Accessibility of Icon Button should be improved

### ISSUE THREAD

**Issue by MarcusMorba** [primefaces/primereact#1014]
### There is no guarantee in receiving an immediate response in GitHub Issue Tracker, If you'd like to secure our response, you may consider *PrimeReact PRO Support* where support is provided within 4 business hours

**I'm submitting a ...**  (check one with "x")

[x] feature request

**Current behavior**
A pure icon button should have an aria-label as the icon itself is not descriptive and accessible like e.g. provided with Material Design Icons

**Expected behavior**
span-tag should have a <span aria-hidden="true"> attribute and <button> an attribute <button aria-label="e.g. search">
 
**Minimal reproduction of the problem with instructions**
Examples & References:
- https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility
- https://material.io/develop/web/components/buttons/

**Please tell us about your environment:**

* **React version:**
n/a

* **PrimeReact version:**
Latest doc example: https://www.primefaces.org/primereact/#/button

* **Browser:** [all ]

* **Language:** [all]

**Comment by sylven32** [comment]
The same issue exists for MenuItem with no label, just an icon: no property allowing to set an aria-label to it.

**Comment by mertsincan** [comment]
Hi all,

@MarcusMorba, Firstly, aria-* (aria-label etc.) properties must be on a focusable element. You can add aria-* properties to Button component. This is supported. For now, I can add aria-hidden="true" to icon tag. If you want to add more features to the icon tag or text tag, I think you can use the custom content option. Exp;
```
<Button aria-label="Test Button" tooltip="This is a custom button">
    <span className="pi pi-home" aria-hidden="true"></span>
    <span style={{marginLeft: '.5rem'}}>Test</span>
</Button>
```
Is it OK for you?

@sylven32, I made some changes in MenuModel API. You can use template options for such issues. Please see;
https://github.com/primefaces/primereact/issues/1783

Best Regards,

**Comment by mertsincan** [comment]
Hi,
I think this issue is resolved.

If the problem persists, please reopen this ticket.
Best Regards,

**Comment by sylven32** [comment]
@mertsincan, cannot find the new template property in the menu item TS definition for 6.0.2.
Is it supposed to work differently? Thanks for your help.

**Comment by mertsincan** [comment]
Hi @sylven32, I'll update all ts definitions for components in the next version. 
Also, please see;
https://primefaces.org/primereact/showcase/#/menumodel ('Template' section)

Best Regards,
